docker/Pro/MAIA-Workspace/Dockerfile

```
@@ -59,6 +59,22 @@ COPY kasmvnc.sh /etc/kasmvnc.sh
 RUN chmod 777 /etc/kasmvnc.sh && \
     chmod a+x /etc/kasmvnc.sh
 COPY supervisord.conf   /etc/supervisord.conf
+
+RUN chmod 777 /etc/
+RUN sudo sed -i 's|a.initSetting("path","websockify")|a.initSetting("path",window.location.pathname.replace(/[^/]*$/,"").substring(1)+"websockify")|' \
```

Hadolint (2.14.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)